app/views/facilities/show.html.haml

```diff
@@ -2,15 +2,17 @@
   = current_facility
 
 - if acting_as? || session_user.try(:operator_of?, current_facility)
-  - instruments = current_facility.instruments.active_plus_hidden
-  - items       = current_facility.items.active_plus_hidden
-  - services    = current_facility.services.active_plus_hidden
-  - bundles     = current_facility.bundles.active_plus_hidden
+  - instruments    = current_facility.instruments.active_plus_hidden
```

Comment:
If we're going to update the style/format of this code, I would lean towards getting rid of all the extraneous space before the `=`. It'll make them ragged, but it'll make diffs cleaner.